client/src/pages/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,154 @@
+          <Typography className={classes.typo}>Settings</Typography>
+        </Grid>
+        <Grid item md={8} sm={12}>
+          <Paper>
+            <Typography variant="h4" className={classes.title}>
+              Edit Profile
+            </Typography>
+            <form className={classes.form} noValidate>
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  FIRST NAME
+                </label>
+                <TextField type="text" id="first-name" placeholder="john" variant="outlined" style={{ width: '70%' }} />
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="last-name" className={classes.label}>
+                  LAST NAME
+                </label>
+                <TextField type="text" id="last-name" placeholder="john" variant="outlined" style={{ width: '70%' }} />
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  GENDER
+                </label>
+                <FormControl variant="outlined">
+                  <Select
+                    labelId="demo-simple-select-outlined-label"
+                    id="demo-simple-select-outlined"
+                    value={gender}
+                    onChange={() => {}}
+                    style={{ width: '100px' }}
+                  >
+                    <MenuItem value={'male'}>Male</MenuItem>
+                    <MenuItem value={'female'}>Female</MenuItem>
+                  </Select>
+                </FormControl>
+              </Box>
+              <Box className={classes.box}>
+                <label htmlFor="first-name" className={classes.label}>
+                  BIRTH DATE
+                </label>
+                <Box component="span" style={{ width: '70%' }}>
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                    <DatePicker
+                      views={['month']}
+                      inputVariant="outlined"
+                      value={selectedDate}
+                      onChange={() => {}}
+                      style={{ width: '100px' }}
+                    />
+                  </MuiPickersUtilsProvider>
+                  <FormControl variant="outlined">
+                    <Select
+                      labelId="demo-simple-select-outlined-label"
+                      id="demo-simple-select-outlined"
+                      style={{ width: '100px' }}
+                    >
+                      {optionSelect}
+                    </Select>
+                  </FormControl>{' '}
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                    <DatePicker
+                      views={['year']}
+                      inputVariant="outlined"
+                      value={selectedDate}
+                      onChange={() => {}}
```

Comment:
not being used? if you're just adding this here because you know in the future you need to do something like this, i would document this with comments with a "TODO" note :)

Reply:
changed to component <Textfield />  type : "date"